samples/bluetooth/df_per_adv_beacon/README.rst

```diff
@@ -0,0 +1,84 @@
+.. _bluetooth_df_per_adv_beacon:
+
+Bluetooth: Direction Finding Periodic Advertising Beacon
+########################################################
+
+Overview
+********
+
+A simple application demonstrating the BLE Direction Finding CTE Broadcaster
+functionality by sending Constant Tone Extension with periodic advertising PDUs.
+
+
+Requirements
+************
+
+* nRF52833DK board with nRF52833 SOC
+* antenna matrix for AoD (optional)
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/bluetooth/df_per_adv_beacon`
+in the Zephyr tree.
+
+There are two build configuration provided with this sample:
+
+* df_aoa.config - enables Angle of Arrival mode
+* df_aod.config - enables Angle of Departure mode (this configuration requires
+  antenna matrix)
+
+By default during build process df_aod.conf configuration is used.
+
+To use Angle of Arrival mode, build this application as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/bluetooth/df_cl_beacon
+   :host-os: unix
+   :board: nrf52833dk_nrf52833
+   :gen-args: DF_MODE=AOA
+   :goals: run
+   :compact:
```

Comment:
Please update this to a single `prj.conf` and then a Kconfig fragment for `AoA` mode.

Reply:
I've updated it as you have requested